### PR TITLE
Added initial batch of flavour text to buttons

### PR DESCRIPTION
**Pull request by blackshadowshade**
Starts to address #1176.

Tested on my local MAMP stack by viewing the button descriptions of Coil (a standard description), Al-Khwarizmi (a description with double quotes present), Zoe (which currently FAILS), Pappy (which has no description)

Requires database update 01176_flavour_text_01.sql

http://jenkins.buttonweavers.com:8080/job/buttonmen-blackshadowshade/848/ (if it passes)

### CODE REVIEW

**Comment by blackshadowshade** [comment]
It will be interesting to see if the 'e' in Zoe renders correctly on the dev site.

**Comment by blackshadowshade** [comment]
http://jenkins.buttonweavers.com:8080/job/buttonmen-blackshadowshade/849/ (if it passes)

**Comment by cgolubi1** [comment]
Test plan:
* the usual update-vs-rebuild test for database changes (important here because the database change is big)
* read through all the lines in the update file and look for typos and problematically bad grammar
* do basically the same test James did where i load a few different buttons on the site, including Zoe, some other buttons with text, and some buttons without text, and make sure they look fine

File: deploy/database/data.button.sql [hunk 1]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
 SET flavor_text='This button gets a different random recipe in each game, with an anime formula'
 WHERE name='RandomBMAnime';
 
+UPDATE button SET flavor_text="Rikachu is a bootleg Button Men created at Origins. He's a Pokedog - from the Pokeman ad opposite the Tess page." WHERE name="Rikachu";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: Nit: s/Button Men/Button Man/

File: deploy/database/data.button.sql [hunk 2]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="ConMan plays games at conventions until (and sometimes past) dawn, drinks coffee like it's water, and he likes to beat people up." WHERE name="ConMan";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: Nit - sentence consistency: s/ and he likes to/ and likes to/

File: deploy/database/data.button.sql [hunk 3]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="The Cheese Weasel is a clever and cunning rodent, bent on quiet world domination. He likes cheese (duh!), any opportunity for eating cheese, stealing cheese...and beating people up if they attempt to come between him and his cheese." WHERE name="Cheese Weasel";
+UPDATE button SET flavor_text="Bennett is a big strong boy who loves to take his time. He enjoys math, videotapes, and beating people up." WHERE name="Bennett";
+UPDATE button SET flavor_text="Chris knows what she likes and she knows how to get it. She likes chemistry, going to the park, and beating people up." WHERE name="Chris";
+UPDATE button SET flavor_text="Darwin is young entrepreneur who enjoys politics, French, and beating people up." WHERE name="Darwin";
+UPDATE button SET flavor_text="Hale is a considerate fellow and excels in every class. He is never too busy to stop and beat people up." WHERE name="Hale";
+UPDATE button SET flavor_text="Morgan knows the rules. She never does anything without reason, and she loves to beat people up." WHERE name="Morgan";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: Nit: i'd say "she never does anything without *a* reason", though the way it's written isn't strictly unparseable.

File: deploy/database/data.button.sql [hunk 4]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="Coil can say the alphabet backwards and sometimes does it without noticing. She likes to be told things she already knows, and she likes to beat people up." WHERE name="Coil";
+UPDATE button SET flavor_text="Crusher is a mean-spirited bruiser with a heart of stone. He likes ikebana, enforcing the whims of callous superiors, and beating people up." WHERE name="Crusher";
+UPDATE button SET flavor_text="Grist is an avid student of Sun Tzu who can move objects with his arms and legs. He likes fast cars, manicures, and beating people up." WHERE name="Grist";
+UPDATE button SET flavor_text="Jellybean is a prominent patent attorney who can think with his spines. He likes collie-spaniel crosses, red ink, and beating people up." WHERE name="Jellybean";
+UPDATE button SET flavor_text="Lucky's father was murdered by an inconsiderate house guest. She likes road movies, vengeance, and beating people up." WHERE name="Lucky";
+UPDATE button SET flavor_text="Peace lives in the exile on the island of St Helena, where she enjoys swing dance, solitaire, and beating people up." WHERE name="Peace";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: s/lives in the exile/lives in exile/

File: deploy/database/data.button.sql [hunk 5]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="One of the best known and most respected mathematicians of the 20th century, Paul Erdos (air-dish) wrote or co-authored 1,475 papers. As he put it, \"A mathematician is a machine for turning coffee into theorems.\"" WHERE name="Paul Erdos";
+UPDATE button SET flavor_text="Pythagoras and his team in Italy called the mathematikoi developed cool triangle theorems, described the five regular solids, and thought that numbers had personal characteristics like male, female, beautiful, and ugly." WHERE name="Pythagoras";
+UPDATE button SET flavor_text="Known informally as \"The Mathemagician,\" Dr. Ron Graham was Chief Scientist at AT&T Labs. He's treasurer of the National Academy of Sciences, and also the ex-president of the International Jugglers Association." WHERE name="Ron Graham";
+UPDATE button SET flavor_text="Theano worked alongside her husband, Pythagoras, writing about mathematics, physics, medicine, and child psychology. She's best known for her work on the \"Golden Mean\" and the \"Golden Rectangle.\"" WHERE name="Theano";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: I think it's a shame that the Button Brains buttons don't mention beating people up.  If you're not up for writing text at this time, let's open a separate issue to fix it.

(I mean, in actual point of fact, the Button Brains buttons do not excel at beating people up.  But i don't think flavor text should be a function of win percentage.)

File: deploy/database/data.button.sql [hunk 6]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="Wastenott is dead, dead, dead, dead. He likes getting out once in a while, talking to old friends and beating people up." WHERE name="Wastenott";
+UPDATE button SET flavor_text="Al-Khwarizmi wrote an amazing book about solving everyday problems in life and business with mathematics. Our word \"algebra\" came from the title of his book, and the term \"algorithm\" is from a Latin translation of his name." WHERE name="Al-Khwarizmi";
+UPDATE button SET flavor_text="Best known for his Theory of Numbers, Carl Friedrich Gauss startled his grade school teacher by deducing that the sum of the integers from 1 to 100 was really the sum of 50 pairs of integers which all add to 101." WHERE name="Carl Friedrich Gauss";
+UPDATE button SET flavor_text="Before becoming the Akami Professor of Internet Mathematics at UC San Diego, Dr. Fan Chung Graham directed math research labs at Bellcore. She and her husband Ron developed semi-random routing theories that keep the Internet running." WHERE name="Fan Chung Graham";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: Fan Chung Graham is in fact the *Akamai* Professor of Internet Mathematics

File: deploy/database/data.button.sql [hunk 7]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="Leonard Eugene Dickson loved pure mathematics. He wrote 267 research papers and 18 books, including the three volume \"History of the Theory of Numbers.\" His favorite quote: \"Thank God that number theory is unsullied by any applications.\"" WHERE name="Leonard Eugene Dickson";
+UPDATE button SET flavor_text="One of the best known and most respected mathematicians of the 20th century, Paul Erdos (air-dish) wrote or co-authored 1,475 papers. As he put it, \"A mathematician is a machine for turning coffee into theorems.\"" WHERE name="Paul Erdos";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: As mentioned on IRC, the o in Erdos has a Hungarumlaut on it.  If you respond to this comment, you too can type Hungarumlaut.

File: deploy/database/data.button.sql [hunk 8]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="Pythagoras and his team in Italy called the mathematikoi developed cool triangle theorems, described the five regular solids, and thought that numbers had personal characteristics like male, female, beautiful, and ugly." WHERE name="Pythagoras";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: Comma Ninja says: "Pythagoras and his team in Italy, called the matematikoi, developed ..."

File: deploy/database/data.button.sql [hunk 9]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="Known informally as \"The Mathemagician,\" Dr. Ron Graham was Chief Scientist at AT&T Labs. He's treasurer of the National Academy of Sciences, and also the ex-president of the International Jugglers Association." WHERE name="Ron Graham";
+UPDATE button SET flavor_text="Theano worked alongside her husband, Pythagoras, writing about mathematics, physics, medicine, and child psychology. She's best known for her work on the \"Golden Mean\" and the \"Golden Rectangle.\"" WHERE name="Theano";
+UPDATE button SET flavor_text="Tirade is a sentient broccoli. He is a master of time space and dimension, and has $50 dollars to spend on candy and gum. He also likes to beat people up." WHERE name="Tirade";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: Pedantic Ninja says: "$50 dollars" should be either "$50" or "50 dollars".

File: deploy/database/data.button.sql [hunk 10]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="The king's loyal nephew, he fights with the strength of three men." WHERE name="Gawaine";
+UPDATE button SET flavor_text="The Queen of the Court of Love, torn between her husband and her lover." WHERE name="Guenever";
+UPDATE button SET flavor_text="King of the Britons, the embodiment of Glory and Honor, and betrayed husband." WHERE name="King Arthur";
+UPDATE button SET flavor_text="The best knight in the world, torn between his love for the queen and duty to his king." WHERE name="Lancelot";
+UPDATE button SET flavor_text="The great magician who advises the king, but who always has his own agenda." WHERE name="Merlin";
+UPDATE button SET flavor_text="The king's bastard son, he has had villainy thrust upon him and revels in it." WHERE name="Mordred";
+UPDATE button SET flavor_text="The dark enchantress who beguiles all with mysterious faerie magic." WHERE name="Morgan Le Fay";
+UPDATE button SET flavor_text="The Lady of the Lake, she holds the powers of Excalibur and Avalon." WHERE name="Nimue";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: As in the case of Button Brains, these buttons ~~are kind of terrible~~ don't actually say "likes to beat people up" in their flavor text, and IMO should.

File: deploy/database/data.button.sql [hunk 11]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="Matt is as loveable as the day is long. Just look at that cute widdow nose. He likes role-playing, the unexpected, and beating people up. (dressed as Hannah)" WHERE name="Matt";
+UPDATE button SET flavor_text="Sven is a Trow (troll) who likes to trade recipes with the other queens at the leather bar, swap housekeeping tips with his neighbors, prune roses, and beat people up." WHERE name="Sven";
+UPDATE button SET flavor_text="Yseulte is a Fuah (bad water faery) who lives by the Maine Avenue fish market. She likes to party with the Grrls, ride vintage Harleys, smear women's makeup, collect scallop shells, and beat people up." WHERE name="Yseulte";
+UPDATE button SET flavor_text="Aldric can belch the anthem of his home town and once drank an entire tankard of beer. He likes damsels, horses, and beating people up." WHERE name="Aldric";
+UPDATE button SET flavor_text="Cesare is the banished heir to the throne of a large kingdom. He likes plotting revenge, procrastinating people, and beating people up." WHERE name="Cesare";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: s/procrastinating people/procrastinating/

File: deploy/database/data.button.sql [hunk 12]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="Chang is far less intelligent than most people think he is. He likes long naps, pieces of string, and beating people up." WHERE name="Chang";
+UPDATE button SET flavor_text="Elihu senses danger at all times, which makes him tense and irritable. His mind is filled with trivial detail, and he loves to beat people up." WHERE name="Elihu";
+UPDATE button SET flavor_text="Farrell spends his weekends by the seaside, watching re-runs and knitting sweaters. But during the week, he likes to beat people up." WHERE name="Farrell";
+UPDATE button SET flavor_text="Frasquito is a very short person and the self-proclaimed inventor of lye. He likes fruit trees and oysters, and he likes to beat people up." WHERE name="Frasquito";
+UPDATE button SET flavor_text="Lark is a triple Mud Baron on the Planet of the Bird People. He likes waffling, collecting shiny objects, and beating people up." WHERE name="Lark";
+UPDATE button SET flavor_text="Luce is a healer and a clever prestidigitator. She can keep children amused for hours, using only a pencil, and she likes to beat people up." WHERE name="Luce";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: Whoa, Comma Ninja actually thinks you should remove a comma.  This is not common.  s/She can keep children amused for hours, using only a pencil/She can keep children amused for hours using only a pencil/

File: deploy/database/data.button.sql [hunk 13]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="Mischa is a tracker whose well-honed nature skills often leads her to the mall. She likes culottes, vegetarian pizza, and beating people up." WHERE name="Mischa";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: "whose well-honed nature skills often *lead* her to the mall

File: deploy/database/data.button.sql [hunk 14]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="Nikolai eat cow, Nikolai smash house with club. Nikolai hate loud noise. Nikolai love beat people up." WHERE name="Nikolai";
+UPDATE button SET flavor_text="Theophilus is a one-eyed giant who knows the future. He is sometimes morose, often melancholy, and always beating people up." WHERE name="Theophilus";
+UPDATE button SET flavor_text="Vincent has the cunning of a fox and table manners of a six-year old boy. He likes tofu, library paste, and beating people up." WHERE name="Vincent";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: Nit: "and *the* table manners of ..." (IMO, anyway)

File: deploy/database/data.button.sql [hunk 15]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="What is love? Only the guy with the big green brain in his hand knows for sure." WHERE name="Max";
+UPDATE button SET flavor_text="Mister Peach teaches old ladies how to dance. He is also creepy." WHERE name="Mister Peach";
+UPDATE button SET flavor_text="The Doctor is in." WHERE name="Simon";
+UPDATE button SET flavor_text="Werner lives underwater and is a crustacean with a helmet. Understandably, people do not trust him." WHERE name="Werner";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: These guys also don't say they like beating people up.

File: deploy/database/data.button.sql [hunk 16]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="Charity wants to be a social worker when she grows up. When not making grouchy faces, she likes to beat people up." WHERE name="Charity";
+UPDATE button SET flavor_text="Chastity wants to be an out-of-work actor when she grows up. She likes her ballet lessons and beating people up." WHERE name="Chastity";
+UPDATE button SET flavor_text="Faith wants to be an eschatologist when she grows up. She likes to watch violent cartoons and beat people up." WHERE name="Faith";
+UPDATE button SET flavor_text="Hope wants to be a navy seal when she grows up. Besides eating her broccoli, she likes to beat people up." WHERE name="Hope";
+UPDATE button SET flavor_text="Prudence wants to be an entomologist when she grows up. She likes chloroforming butterflies and beating people up." WHERE name="Prudence";
+UPDATE button SET flavor_text="Temperance wants to be a postal worker when she grows up. She likes small defenseless animals and beating people up." WHERE name="Temperance";
+UPDATE button SET flavor_text="Spells and mystic zaps are what Artax do best. His two best words of power are Abracadabra and beatpeopleup." WHERE name="Artax";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: I don't like this sentence, so i'm not 100% sure either way, but i think it should be "...what Artax *does* best"

File: deploy/database/data.button.sql [hunk 17]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="Sadakuro owns nothing but a good book and the deadliest fighting tools. He likes long walks in the dark, remaining perfectly still, and beating people up." WHERE name="Sadakuro";
+UPDATE button SET flavor_text="Tonase picks a fight with everyone she meets. She is opinionated and naive, she adores small dogs and furniture, and she likes to beat people up." WHERE name="Tonase";
+UPDATE button SET flavor_text="Yuranosuke is a fierce warrior who takes nothing for granted. His highest aspiration is to serve the forces of good, and he likes to beat people up." WHERE name="Yuranosuke";
+UPDATE button SET flavor_text="Aylee thinks she looks like a pretty flower. She attended Cannibals Anonymous, and she likes to eat people up." WHERE name="Aylee";
+UPDATE button SET flavor_text="Bun-Bun isn't your typical mini-lop rabbit. He hates telemarketers with a passion and frequently beats people up." WHERE name="Bun-Bun";
+UPDATE button SET flavor_text="KiKi is the friendliest ferret around. She loves lights, and games, and candy, and never means to beat people up." WHERE name="Kiki";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: I think "Kiki" not "KiKi", and wikipedia agrees.

File: deploy/database/data.button.sql [hunk 18]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="Riff is a freelance inventor whose desire to build nifty toys is exceeded only by his joy in building big guns (which sometimes beat people up)." WHERE name="Riff";
+UPDATE button SET flavor_text="Torg is a freelance webpage designer. He makes occasional trips to alternate dimensions or past history, where he sometimes beat people up." WHERE name="Torg";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: "...where he sometimes *beats* people up"

File: deploy/database/data.button.sql [hunk 19]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="Clare is an international spelling and thumb wrestling champion. She enjoys hiking, fishing, and beating people up." WHERE name="Clare";
+UPDATE button SET flavor_text="Hammer is a vicious, sweaty man with a garbage can on his head, and he likes to beat people up." WHERE name="Hammer";
+UPDATE button SET flavor_text="Hannah clings to a deep and abiding faith in the fundamental goodness of all human beings, and she likes to beat people up." WHERE name="Hannah";
+UPDATE button SET flavor_text="Iago is a sorcerer with mystic powers and is available for bar mitzvahs. He also likes to beat people up." WHERE name="Iago";
+UPDATE button SET flavor_text="Karl is a world champion kickboxer and a pathological liar. He likes playing drums, and beating people up." WHERE name="Karl";
+UPDATE button SET flavor_text="Kith enjoys travel and nature photography. She binds books in her spare time, and she likes to beat people up." WHERE name="Kith";
+UPDATE button SET flavor_text="Kublai is an embittered fast-food worker whose time is never his own. He likes malts, orangutangs, and beating people up." WHERE name="Kublai";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: Should be "orangutans"

File: deploy/database/data.button.sql [hunk 20]
```diff
@@ -1152,3 +1152,315 @@ UPDATE button
...
+UPDATE button SET flavor_text="Buddy is a heartless brute without soul or conscience. It likes cabbages and howling, and it likes to beat people up." WHERE name="Buddy";
+UPDATE button SET flavor_text="Dunkirk is a mysterious Balkan folk dancer. He often taunts his victims with hours of tuneless humming, and he likes to beat people up." WHERE name="Dunkirk";
+UPDATE button SET flavor_text="Durban McGinty sleeps with one eye open and is easily startled when awake. He is never without something to chew on, and he likes to beat people up." WHERE name="McGinty";
+UPDATE button SET flavor_text="Starchylde is a ferocious vampire hunter and a prolific poet. She likes garlic, mice and ultraviolet light, and she likes to beat people up." WHERE name="Starchylde";
+UPDATE button SET flavor_text="Tiffany is a robotics expert and gourmet chef. Her travels often take her to remote locations in Tibet and Pakistan, and she likes to beat people up." WHERE name="Tiffany";
+UPDATE button SET flavor_text="Rikachu is a bootleg Button Men created at Origins. He's a Pokedog - from the Pokeman ad opposite the Tess page." WHERE name="Rikachu";
```
Comment (cgolubi1) [inline deploy/database/data.button.sql]: Should be "Button *Man*"

**Comment by cgolubi1** [comment]
Oops, realised i'd gotten to Rikachu a second time.  Looks like your database updates file contains each entry twice:
```bash
$ wc deploy/database/updates/01176_flavour_text_01.sql 
     312    9190   61674 deploy/database/updates/01176_flavour_text_01.sql
$ sort deploy/database/updates/01176_flavour_text_01.sql | uniq | wc
     156    4595   30837
```

**Comment by cgolubi1** [comment]
Anyway, throwing this back over the fence to address the things noticed so far, and then i'll take another look and do the other tests.

**Comment by blackshadowshade** [comment]
So, I've made all the minor changes that you've suggested. I'm not entirely sold on changing the flavour text to include beating people up when it doesn't already have it, so I'd love to see what the testers and players think about this.

http://jenkins.buttonweavers.com:8080/job/buttonmen-blackshadowshade/851/ (if it passes)

**Comment by cgolubi1** [comment]
Not sure what you tried with the Zoe flavor text, but in my vagrant instance, it doesn't work at all, and i think that's a pretty good predictor of what will happen on the live site.  Does this work for you?  Here's what i see:

zoe_flavor_text

**Comment by cgolubi1** [comment]
Also, i feel like i've mentioned the Paul Erdős umlaut thing a couple of times --- are you planning to snag that as well along with the Zoe fixes, or not at this time?

Those are the only two things i see.

**Comment by irilyth** [comment]
About "and likes to beat people up", I'm on the fence. I like having it; but if someone wrote flavor text for one or more buttons they designed, deliberately left it off, and doesn't want it on there, perhaps we shouldn't include it. But how would we even know? Meh.

**Comment by cgolubi1** [comment]
Yeah, in the end i don't think i feel strongly about "and likes to beat people up" --- a few of the sets appear not to have it, and i think their flavor text isn't as good as the flavor text that does, but that's life.  I'm inclined to agree with shadowshade that we shouldn't mess with it right now.

**Comment by blackshadowshade** [comment]
Chaos, about Zoe and Paul Erdos, I'm thinking that we might actually need you to submit a small pull request putting the right Unicode characters into the SQL after this pull request has been accepted. The way that I am able to get the right character on my local MAMP stack is to post to the forum, and then copy the resulting encoded characters from the forum post in the database.

I tried doing the same from the vagrant instance on dev, but it's quite possible that the terminal that I used changed the characters somehow.

**Comment by cgolubi1** [comment]
That doesn't make a lot of sense to me --- i think we want the database as stored to work portably, so someone is going to have to figure out how to configure UTF-8 (or something) in our database settings for real, e.g. something along the lines of http://cameronyule.com/2008/07/configuring-mysql-to-use-utf-8/ or http://mzsanford.com/blog/mysql-and-unicode/ (but the latter assumes you already know what's going on and have just hit the error that person hit, and the former doesn't really say what you have when you're done).  What we *really* want, IMO, is a way to use an ascii representation of unicode characters (e.g. `\xc3\xa4` for a-umlaut) in the database, and fix the database settings so that it gets loaded as Unicode.

If someone wants to figure out how to do that portably right now, great.  If not, though, i don't think it's a good idea to add stuff to our database that we know is not portable.  So maybe the short-term solution is just to leave Zoe's umlauts out for now.

**Comment by blackshadowshade** [comment]
Okay, I've taken the umlauts off Zoe's name for the moment.

http://jenkins.buttonweavers.com:8080/job/buttonmen-blackshadowshade/854/ (if it passes)

**Comment by cgolubi1** [comment]
Opened #1699 for the unicode issue.

**Comment by cgolubi1** [comment]
Works for me.  Merging.

**Pull request merged**